core/utils_test.py

```diff
@@ -892,7 +892,7 @@ def test_generate_new_session_id(self) -> None:
     def test_require_valid_name_with_incorrect_input(self) -> None:
         with self.assertRaisesRegex(
             utils.ValidationError,
-            'The length of the exploration title should be between 1 and 50 ' 'characters; received '):   # pylint: disable=line-too-long
+            'The length of the exploration title should be between 1 and 50 characters; received '):   # pylint: disable=line-too-long
```

Comment:
I think you don't need the line-too-long pragma if you just split into two lines (i.e. keep the quotes that were deleted and put a newline after the first deleted quote.

Reply:
done